onnxmltools/convert/sparkml/convert.py

```diff
@@ -5,6 +5,7 @@
 from ..common.onnx_ex import get_maximum_opset_supported
 from ..common._topology import convert_topology
 from ._parse import parse_sparkml
+from . import operator_converters
```

Comment:
This is needed or else this error will fail on Spark GBT conversion: `ValueError: Unsupported shape calculation for operator pyspark.ml.classification.GBTClassificationModel`

Reply:
Ok. It is weird we did not catch this error until now.

Reply:
Yeah this was a weird one. Because when I do a `pip3 install` on the remote onnxmltools, it throws the `AnalysisException: Unable to infer schema for Parquet. It must be specified manually.` error that i opened up the issue #648 for. When we forked the repo to put up a fix, the `unsupported shape calculation error` threw until we added in the import. We assumed that it was due to Spark capabilities being still built or or potentially it gets added in when packaged? But yeah weird error